package.json

```diff
@@ -31,7 +31,6 @@
     "test:nightly": "yarn test:browser --grep @nightly",
     "test:fullfunctional": "yarn test:nightly",
     "test:smoke": "cross-env NODE_PATH=. NODE_ENV=test TS_NODE_PROJECT=app/server/tsconfig.json mocha test/smoke/*.ts --grep @smoke --colors --reporter mochawesome  --reporter-options reportDir=smoke-output/reports/smoke,inlineAssets=true --exit",
-    "test:audit": "./bin/run-yarn-audit.sh -f yarn-audit-known-issues",
```

Comment:
is the yarn-audit-known-issues file not needed anymore?

Reply:
We still need yarn-audit-known-issues for the suppressions but we don't use run-yarn-audit.sh anymore